pkg/api/registry/registry.proto

```diff
@@ -22,61 +22,40 @@ message Destination {
     uint32 weight = 2;
 }
 
-message NetworkServiceManager {
-    string name = 1;
-    string url = 2;
-    google.protobuf.Timestamp expiration_time = 3;
-    string state = 4;
+message NetworkServiceLabels {
+    map<string, string> labels = 1;
 }
 
 message NetworkServiceEndpoint {
     string name = 1;
-    string payload = 2;
-    string network_service_name = 3;
-    string network_service_manager_name = 4;
-    map<string, string> labels = 5;
-    string state = 6;
-
+    repeated string network_service_name = 2;
+    map<string, NetworkServiceLabels> network_service_labels = 3;
     // Endpoint callback address, could be any of unix/tcp/callback schema ones.
-    string url = 7;
-}
-
-message FindNetworkServiceRequest {
-    string network_service_name = 1;
-}
-
-message FindNetworkServiceResponse {
-    string payload = 1;
-    NetworkService network_service = 2;
-    map<string, NetworkServiceManager> network_service_managers = 3;
-    repeated NetworkServiceEndpoint network_service_endpoints = 4;
+    string url = 4;
+    // Network service manager URL for the endpoint.
+    string network_service_manger_url = 5;
```

Comment:
Do we need url and network_service_manager_url?

Is it possible to get away with using one url or do we need both?

Reply:
> Do we need url and network_service_manager_url?
Yes, we need both URLs
As I understand: 
`url` is using for callback goals.
`network_service_manger_url` is planned to use for different goals such as connecting to remote nsmgr, discover all endpoints related to the specific manager and etc.

Reply:
Yeah... I still don't see why we need two urls.  If you want callback goals (ie, localbypass) ... that's data for use locally inside the K8s NSMgr.  It shouldn't leak out to the registry.

Reply:
ok, we've little bit discussed this moment with @haiodo and looks like we can remove one of URLs.

Reply:
Fixed https://github.com/networkservicemesh/api/pull/41/commits/3b53da4e4e1a26185f782b92ce1ca044b516c38a